plugins/permissions/src/interactions/commands.ts

```diff
@@ -39,7 +39,7 @@ export function validateApplicationCommandOptions(
       throw new Error("Invalid application command description.");
     }
 
-    option.choices?.every((choice) => {
+    if ("choices" in option) option.choices?.every((choice) => {
```

Comment:
Please add an if block to avoid confusion

Reply:
This is not resolved?

Reply:
> This is not resolved?

sorry!, it's solved now